manifest/spark-operator-with-webhook.yaml

```diff
@@ -57,6 +57,30 @@ spec:
         - -logtostderr
         - -enable-webhook=true
 ---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: sparkoperator-init
+  namespace: spark-operator
+  labels:
+    app.kubernetes.io/name: sparkoperator
+    app.kubernetes.io/version: v2.3.1-v1alpha1
+spec:
+  backoffLimit: 3
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: sparkoperator
+        app.kubernetes.io/version: v2.3.1-v1alpha1
+    spec:
+      serviceAccountName: sparkoperator
+      restartPolicy: Never
+      containers:
+      - name: main
+        image: gcr.io/spark-operator/spark-operator:v2.3.1-v1alpha1-latest
+        imagePullPolicy: Always
+        command: ["/usr/bin/gencerts.sh", "-p"]
```

Comment:
This requires that the image has newest hack version populated (with new namespace)

Reply:
Will do.